components/Form/index.js

```diff
@@ -37,68 +26,68 @@ export default function Form() {
   const handleSubmit = (event) => {
     event.preventDefault();
 
-    const formData = new FormData(event.target);
-    const data = Object.fromEntries(formData);
-
-    //neues Projekt-Objekt erstellen
+    //create new project-object
     const project = {
       name: event.target.projectname.value,
       items: items,
+      id: uuidv4(),
     };
 
     const resetItems = partList.map((item) => {
       return { ...item, price: 0, value: "" };
     });
 
-    //das neue Projekt zur Liste der Produkte hinzufügen
-    setProjects([...projects, project]);
+    //add new project to list
+    addProject(project);
```

Comment:
usually the comments should be sparse. in his case the code already says: addProject, so it is clear what is done here.

Reply:
this comment is also in german in an english code...so it was a notice for me to learn, but thx for remember🦄